flow/endpoint.yaml

```diff
@@ -79,8 +79,8 @@ components:
           - '/components/schemas/Device.Ethernet/properties/name'
           - '/components/schemas/Device.Ipv4/properties/name'
           - '/components/schemas/Device.Ipv6/properties/name'
-          - '/components/schemas/Device.Bgpv4RouteRange/properties/name'
-          - '/components/schemas/Device.Bgpv6RouteRange/properties/name'
+          - '/components/schemas/Bgp.V4RouteRange/properties/name'
+          - '/components/schemas/Bgp.V6RouteRange/properties/name'
           example: ['Eth 1', 'Eth 2', 'Eth 3']
```

Comment:
add a couple of examples of names of v4routerange/v6route_range.

Reply:
Done

Reply:
Also : snappi code example given:
#Example of accessing BGP route ranges from Flow End Point.
flow1 = config.flows.flow(name='f1')[-1]
flow1.tx_rx.device.tx_names = [p1_v4rr.name, p1_v6rr.name]
flow1.tx_rx.device.rx_names = [p2_v4rr.name, p2_v6rr.name]